app/assets/stylesheets/petitions/views/_home.scss

```diff
@@ -56,11 +56,19 @@
 .threshold-panel:hover, .threshold-panel:focus {
   background-color: darken($petitions-panel-colour, 5%);
 }
-
 .response-threshold, .debate-threshold {
   background-position: 15px 15px;
   background-repeat: no-repeat;
 }
+.pull-quote {
```

Comment:
Can we wrap `.pull-quote` and `.response-intro` inside a `.threshold-petitions` so it is more specific? There is no obvious relationship here and these will apply across the site.